aiohttp/client.py

```diff
@@ -351,6 +360,9 @@ async def _request(
         proxy_headers: Optional[LooseHeaders] = None,
         trace_request_ctx: Optional[SimpleNamespace] = None,
         read_bufsize: Optional[int] = None,
+        max_line_size: Optional[int] = 8190,
+        max_headers: Optional[int] = 32768,
+        max_field_size: Optional[int] = 8190,
```

Comment:
Shouldn't these default to `None`? This currently means that the session defaults are only used when the user explicitly passes `None`, so they would need to make every call like `client.request(..., max_line_size=None, max_headers=None, max_field_size=None)`, making the session defaults pointless.

Reply:
Thanks for the review. Of course that's right. I changed it accordingly and added a couple of tests.